instaclustr/resource_cluster.go

```diff
@@ -687,16 +723,35 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	if len(cluster.DataCentres[0].ResizeTargetNodeSize) > 0 {
 		nodeSize = cluster.DataCentres[0].ResizeTargetNodeSize
 	}
+	// set data centre
+	if cluster.DataCentre != "" {
+		d.Set("data_centre", cluster.DataCentre)
+	} else if len(cluster.DataCentres) == 1 {
+		d.Set("data_centre", cluster.DataCentres[0].Name)
```

Comment:
[question] Why are we setting this to DataCentres[0]? Seems like this should be nil or "" and then get assigned below in DataCentres?